vcpkg/commands/create.md

```diff
@@ -0,0 +1,158 @@
+---
+title: vcpkg create
+description: Command line reference for the vcpkg create command. The `vcpkg create` command is used to generate a new vcpkg port from a specified URL.
+ms.date: 08/01/2023
+---
+
+# vcpkg create
+
+## Synopsis
+
+```no-highlight
+vcpkg create <port-name> <url-to-source> [archive-file-name]
+```
+
+## Description
+
+The `vcpkg create` command is a tool that facilitates the generation of new ports for libraries within the vcpkg ports tree. To do this, it requires a port name and a URL that points to the source code of the library in question.
+
+Upon execution, this command initiates a download of the source code from the provided URL. It then creates an initial "portfile.cmake" and "vcpkg.json" in a newly formed directory.
```

Comment:
Elsewhere, filenames are styled in monospace. I'm not sure what is the most common style across all the documentation, but we should stick to one.